e2e/keywords/common.resource

```diff
@@ -82,12 +82,11 @@ Enable v2 data engine and add block disks
 
 Init worker node variables
     ${worker_nodes}=    get_worker_nodes
-    Set Test Variable    ${NODE_0}    ${worker_nodes[0]}
-    Set Test Variable    ${NODE_1}    ${worker_nodes[1]}
-    Set Test Variable    ${NODE_2}    ${worker_nodes[2]}
+    Run Keyword And Ignore Error    Set Test Variable    ${NODE_0}    ${worker_nodes[0]}
+    Run Keyword And Ignore Error    Set Test Variable    ${NODE_1}    ${worker_nodes[1]}
+    Run Keyword And Ignore Error    Set Test Variable    ${NODE_2}    ${worker_nodes[2]}
```

Comment:
After review, the changes LGTM. 

I have one question. Why do we need to use `Run Keyword And Ignore Error` here? Have you encountered this issue multiple times? 

If so, it might be something we should be aware of and monitor carefully in the future.